python/ray/_private/thirdparty/pynvml/__init__.py

```diff
@@ -1,3 +1,3 @@
 from ray._private.thirdparty.pynvml.pynvml import *
-# current version
-__version__ = "12.535.133"
\ No newline at end of file
+# nvdia-ml-py version
```

Comment:
Add a comment saying why we pick this version: something like we are using this version because it uses v2 api and supports a wider range of drivers.